Modifiqué el archivo index

charo007 opened the pull request:
Hice modificaciones al archivo index, estas modificaciones se
basaron en agregar un carrusel de imágenes para hacer que el sitio
sea más atractivo para el usuario.

<!-- Saludos, si usted esta leyendo esto, significa que quiere colaborar con el proyecto aportando quizás mejoras o información que encontró mientras utilizo el mismo, es por eso que queremos agradecerle que se tome su tiempo por hacer lo antes mencionado. -->

<!-- 
	Utilice la siguiente forma de títulos-
•	Si es una mejora en la base de datos [DB] titulo
•	Si es una mejora de estilos [CSS] titulo
•	Si es una mejora en el código [HTML] titulo
Si utiliza varias mejoras, no especifique nada, solo el titulo.

-->
HTML
<!-- Necesitamos el que titulo del tema sea lo mas representativo posible. -->

##### ¿QUÉ CAMBIO PROPONE?:
<!--
	Agregue un pequeño texto que explique cual es el bug que desea reportar o cual es la situación de mejora que le gustaría que apliquemos. 
-->
Agregue un carrusel con imágenes al index del proyecto para que los usuarios encuentren más atractivo el sitio. Además, cada imagen contiene una pequeña descripción de cada sección del mismo.


##### NOTAS EXTRA:
<!--
Cualquier información que pueda ayudar a los desarrolladores a identificar y solucionar el problema debe ser puesta aquí.
-->

##### BRANCH(ES):
<!-- 
Especifique la(s) rama(s) afectada(s) por este problema: master, 0.x, 1.x, u otra rama. 
Reemplace el valor de master por la rama correspondiente.
-->

master



##### OTRAS PERSONALIZACIONES:
<!-- 
En caso de que se produzca alguna mejora en el código, avísennos, caso contrario, no es necesario que usted escriba nada aquí. (solo para programadores)
-->




<!-- ------------------------- THE END ------------------------------
 Gracias nuevamente por su colaboración, atenderemos su tema lo antes posible. 
-->

charo007 committed:
Modifiqué el archivo index

Hice modificaciones al archivo index, estas modificaciones se
basaron en agregar un carrusel de imágenes para hacer que el sitio
sea más atractivo para el usuario.

RainTrap reviewed (changes requested):
El código esta bien, un detalle, dentro de los div de la linea 49, 56 y 63 agregar un botón el cual direccione automáticamente el usuario a la pagina correspondiente.

charo007 committed:
Archivo index.css

Tuve un problema con las ramas en los primeros cambios del trabajo.

charo007 committed:
Agregando Botón en el carrusel del Index.

En este cambio estoy agregando el cambio solicitado por el
admnistrador sobre el index.

RainTrap reviewed (approved):
Gracias por realizar las modificaciones correspondientes, el código esta bien.
